<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>731</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>971</width>
     <height>641</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>26</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideRight</enum>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="NFA">
    <attribute name="title">
     <string>NFA</string>
    </attribute>
    <widget class="QTableWidget" name="nfaTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>981</width>
       <height>591</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="DFA">
    <attribute name="title">
     <string>DFA</string>
    </attribute>
    <widget class="QTableWidget" name="dfaTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>981</width>
       <height>591</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="MinDFA">
    <attribute name="title">
     <string>MinDFA</string>
    </attribute>
    <widget class="QTableWidget" name="minDfaTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>981</width>
       <height>591</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ToCode">
    <attribute name="title">
     <string>Code</string>
    </attribute>
    <widget class="QTextBrowser" name="codeBrowser">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>1</y>
       <width>981</width>
       <height>591</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>输入正则表达式:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>561</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>(a|b)*cde</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>转换</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
